mail/notmuch/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           conflicts_build 1.0
 
 name                notmuch
-version             0.28
+version             0.28.3
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Indeed. I removed the line, 0 is the default anyway.